cves/CVE-2013-1896.yml

```diff
@@ -171,8 +204,11 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
+      Anything can be passed in the body of an HTTP request, and one of the most
+      important features of an HTTP server is to ensure that the infomormation is
+      sanatized, or filtered to ensure that only real requests are made.
```

Comment:
You should also mention that servers should check for non-native HTTP requests (like those part of WebDAV) and find a way to handle such requests

Reply:
Added a bit more about non-native HTTP requests